### PR TITLE
LPS-162114 [EVP] Create NEWS Section for Home page

### PR DESCRIPTION
**Pull request by JoMaAlves**

### CODE REVIEW

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:sf

**Review by juliocezarjc — APPROVED** [review]
LGTM ! 🔥

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/news/index.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	border: 1px solid;
+	border-radius: 8px;
+}
+
+.evp_news_container img {
+	height: 24px;
+	width: 24px;
+}
+
+.evp_news_container .news-content .news-text {
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/news/index.css:11]: Please, change the class name here.
```suggestion
.evp-news-container .news-content .news-text {
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/news/index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<div class="border-neutral-3 d-flex evp_news_container p-3">
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/news/index.html:1]: Please, change the class name here.
```suggestion
<div class="border-neutral-3 d-flex evp-news-container p-3">
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/slider-evp/configuration.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+	"fieldSets": [
+		{
+			"fields": [
+				{
+					"dataType": "int",
+					"defaultValue": "3",
+					"label": "number-of-slides",
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/slider-evp/configuration.json:8]: Please, change the label name here.
```suggestion
					"label": "Number of slides",
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/slider-evp/index.js [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+				else {
+					move(MOVE_RIGHT, index);
+				}
+			}
+
+			clearInterval(intervalId);
+			intervalId = createInterval();
+		});
+	});
+})();
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/slider-evp/index.js:130]: Please, change it.

```suggestion
function main() {
	let intervalId = createInterval();

	if (this.nextItemIndex && this.nextItemIndex < items.length) {
		const activeItem = fragmentElement.querySelector(
			'.carousel-item.active'
		);
		const nextItem = items[this.nextItemIndex];

		activateIndicator(activeItem, nextItem);
		activateItem(activeItem, nextItem);
	}

	indicators.forEach((indicator, index) => {
		indicator.addEventListener('click', () => {
			const indexActiveIndicator = indicators.indexOf(
				getActiveIndicator()
			);

			if (index !== indexActiveIndicator) {
				if (index < indexActiveIndicator) {
					move(MOVE_LEFT, index);
				}
				else {
					move(MOVE_RIGHT, index);
				}
			}

			clearInterval(intervalId);
			intervalId = createInterval();
		});
	});
};

main();
```

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some suggestions.

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:relevant

**Review by juliocezarjc — APPROVED** [review]
LGTM !

**Review by silasbsi — APPROVED** [review]

**Comment by silasbsi** [comment]
ci:forward

**Comment by JoMaAlves** [comment]
ci:forward

**Pull request closed without merging**